frontend/src/components/eMIB/TeamInformation.jsx

```diff
@@ -226,7 +244,6 @@ class TeamInformation extends Component {
               onClick={this.openPopup}
               className="btn btn-secondary"
               style={styles.button}
-              aria-label={LOCALIZE.emibTest.background.teamInformation.teamChart.ariaLabel}
```

Comment:
Same for this label

Reply:
Added the label back in commit #https://github.com/code-for-canada/project-thundercat/pull/438/commits/2796bb97ad19d5754454e2717e3c0f96d9ce9b7e